src/TitleDetails/AddPieceModal/AddPieceModalContainer.js

```diff
@@ -32,26 +51,38 @@ const AddPieceModalContainer = ({
     ? PIECE_FORMAT_OPTIONS.filter(({ value }) => [PIECE_FORMAT.electronic, PIECE_FORMAT.physical].includes(value))
     : PIECE_FORMAT_OPTIONS.filter(({ value }) => value === initialValues.format);
 
+  if (!locations) return null;
+
   return (
     <AddPieceModal
       close={close}
       createInventoryValues={createInventoryValues}
       initialValues={initialValues}
       instanceId={instanceId}
+      locationIds={locationIds}
+      locations={locations}
       onCheckIn={onCheckIn}
       onSubmit={onSubmit}
       pieceFormatOptions={pieceFormatOptions}
     />
   );
 };
 
+AddPieceModalContainer.manifest = Object.freeze({
+  pieceLocations: {
```

Comment:
`pieceLocations` doesn't reflect the containing request. I would say it's just `locations`.